pkg/domain/entities/container_ps.go

```diff
@@ -14,7 +15,7 @@ type ListContainer struct {
 	// Container command
 	Command []string
 	// Container creation time
-	Created int64
+	Created time.Time
```

Comment:
@rhatdan @jwhonce Is this going over the wire? As in, is this a breaking change to our JSON format?

Reply:
I don't believe this is going over the wire.  The time.Time came over the wire though.

Reply:
But this may be a breaking change for anyone who has vendored in /pkg/ directories. So we might have to wait to fix this until 3.0.